defaults/main.yml

```diff
@@ -33,6 +47,19 @@ OPENCRAFT_ENV_TOKENS:
   SECRET_KEY: 'your-random-secret-key'
   WATCH_ORGANIZATION: 'your-github-organization'
 
+# These env tokes are generated from other ansible variables, entries in
+# this dictionary can be overriden by OPENCRAFT_ENV_TOKENS
+OPENCRAFT_GENERATED_ENV_TOKENS:
```

Comment:
I'm not sure I understand why this separate dict is necessary.  Why not just append these variables directly to `OPENCRAFT_ENV_TOKENS`, and have a single dict, instead of having to merge them together later?